dipy/direction/peaks.py

```diff
@@ -572,3 +571,49 @@ def reshape_peaks_for_visualization(peaks):
         peaks = peaks.peak_dirs
 
     return peaks.reshape(np.append(peaks.shape[:-2], -1)).astype('float32')
+
+
+def peaks_from_positions(positions, odfs, sphere, relative_peak_threshold=.5,
+                         min_separation_angle=25, is_symmetric=True, npeaks=5):
```

Comment:
We start to introduce keyword-only arguments in the codebase
Can you replace this by:

`def peaks_from_positions(positions, odfs, sphere, *, relative_peak_threshold=.5,
                         min_separation_angle=25, is_symmetric=True, npeaks=5):`

Reply:
done.

Reply:
it does not appear here as done, so might have done a mistake in your push.